views/admin/javascripts/exhibits.js

```diff
@@ -468,4 +475,229 @@ Omeka.ExhibitBuilder = {};
+            file_id = $('input[name="file_id"]:checked').val();
+            event.preventDefault();
+            chooseCoverImage(file_id);
+            coverImagePanel.dialog('close');
+        });
+
+        $('#cover-image-item-options').on('click','.file-select .item-file', function(event) {
+            $(this).find('input[type="radio"]').prop('checked', true);
+            $('.selected').removeClass('selected');
+            $(this).addClass('selected');
+        });
+
+        $('#change-selected-item').on('click', function (event) {
+            event.preventDefault();
+            $('#cover-image-panel')
+                .removeClass('editing-cover-image')
+                .addClass('editing-selection');
+        });
+
+        $('#revert-selected-item').on('click', function (event) {
+            event.preventDefault();
+            $('#cover-image-panel')
+                .removeClass('editing-selection')
+                .addClass('editing-cover-image');
+        });
+    }
+
+    Omeka.ExhibitBuilder.setUpCoverImageSelect = function(browseUri) {
+        /*
+         * Use AJAX to retrieve the list of items that can be attached.
+         */
+        function getItems(uri, parameters) {
+            $('#cover-image-panel').addClass('loading');
+            $.ajax({
+                url: uri,
+                data: parameters,
+                dataType: 'html',
+                method: 'GET',
+                success: function(data) {
+                    $('#item-select').html(data);
+                },
+                error: function(xhr, textStatus, errorThrown) {
+                    alert('Error getting items: ' . textStatus);
+                },
+                complete: function() {
+                    $('#cover-image-panel').removeClass('loading');
+                }
+            });
+        }
+
+        getItems(browseUri);
+
+        // Make search and pagination use AJAX to respond.
+        $('#search').submit(function(event) {
+            event.preventDefault();
+            getItems(this.action, $(this).serialize());
+            Omeka.ExhibitBuilder.setSearchVisibility(false);
+        });
+        $('#item-form').on('click', '.pagination a, #view-all-items', function (event) {
+            event.preventDefault();
+            getItems(this.href);
+            Omeka.ExhibitBuilder.setSearchVisibility(false);
+        });
+        $('#item-select').on('submit', '.pagination form', function (event) {
+            event.preventDefault();
+            getItems(this.action + '?' + $(this).serialize());
+            Omeka.ExhibitBuilder.setSearchVisibility(false);
+        });
+
+        Omeka.ExhibitBuilder.setSearchVisibility(false);
+        $('#show-or-hide-search').click(function (event) {
+            event.preventDefault();
+            Omeka.ExhibitBuilder.setSearchVisibility();
+        });
+
+        $('#exhibit-page').on('change', function(event){
```

Comment:
Something went odd with the comment, but these two jQuery calls also look like leftovers